src/lib/libdylink.js

```diff
@@ -1187,21 +1187,19 @@ var LibraryDylink = {
 #if DYLINK_DEBUG
     dbg('loadDylibs:', dynamicLibraries);
 #endif
+    addRunDependency('loadDylibs');
 
     // Load binaries asynchronously
-    addRunDependency('loadDylibs');
-    dynamicLibraries
-      .reduce((chain, lib) => chain.then(() =>
-        loadDynamicLibrary(lib, {loadAsync: true, global: true, nodelete: true, allowUndefined: true})
-      ), Promise.resolve())
-      .then(() => {
-        // we got them all, wonderful
-        reportUndefinedSymbols();
-        removeRunDependency('loadDylibs');
-  #if DYLINK_DEBUG
-        dbg('loadDylibs done!');
-  #endif
-      });
+    for (var lib of dynamicLibraries) {
```

Comment:
I wonder if this would benefit from Promise.all? Is it safe to load them in parallel?

Reply:
No, we have to load them in the given order I believe.

I thought there was a commend about this, but I don't see it now.